orion/evaluation/utils.py

```diff
@@ -5,12 +5,16 @@
 
 
 def from_pandas_contextual(df):
-    """ convert contextual `pd.DataFrame` to list of tuples.
+    """ Convert contextual ``pandas.DataFrame`` to list of tuples.
 
     Args:
-        * df (pd.DataFrame): contains start and end.
-    """
+        df (DataFrame): anomalies, passed as ``pandas.DataFrame``
+        containing two columns: start and stop.
 
+    Returns:
+        list:
+            tuple (start, end) timestamp.
```

Comment:
I would make this description a bit more verbose, otherwise it can be confusing.
Something like:

```
list:
    list of tuples containing start and end timestamps.
```

Reply:
Also, since the method has an explicit `rise` within it, it would be great to add a `Rises` entry indicating it.

Reply:
Updated.